src/imagery/i.eodag/i.eodag.py

```diff
@@ -65,7 +65,7 @@
 # % key: producttype
 # % type: string
 # % description: Imagery product type to search for
-# % required: no
+# % required: yes
```

Comment:
Not 100% sure what the right choice is here, as when you want to list available product types, you would not specify that option here...
You could leave it as not required, but use rules to allow or require certain combinations of options and flags: https://grass.osgeo.org/grass83/manuals/g.parser.html#conditional-parameters
Just something to consider...

Reply:
I put it back to **no**. Couldn't add the rules though as it will cause problems, esp. with the the file option when the file is referring to a geojson and not a text file. Instead, I added a warning that the producttype is not set, when it is expected to be preferably set.